test/collections/HashDictionaryTest.ooc

```diff
@@ -98,43 +97,41 @@ HashDictionaryTest: class extends Fixture {
 			expect(dictionary2 get("First", "Default") == "First", is true)
 			expect(dictionary2 get("Int", 0) == 1, is true)
 		})
-
 		this add("Get from primitive", func {
 			dictionary := HashDictionary new()
 			dictionary add("First", 1337)
-			expect(dictionary getAsType("First", Int) == 1337)
-			expect(dictionary getAsType("First", String) == null)
-			expect(dictionary getAsType("Second", Int) == null)
-			expect(dictionary getAsType("First", Cell<TestCover>) == null)
-			expect(dictionary getAsType("First", TestClass) == null)
+			dictionary add("First2", TestClass new(1337, "String"))
```

Comment:
`First2` doesn't make much sense. How about `Third`, or replace `Second` below with `Third` and `First2` with `Second`?